clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2733,7 +2739,11 @@ void pollStatusUntilComplete(
         }
 
         // Every known datacenter have successfully reported a completed status at least once.
-        LOGGER.info("Successfully pushed {}", topicInfo.topic);
+        if (isTargetedRegionPush) {
```

Comment:
Can we do one-liner for this?
```
LOGGER.info("Successfully pushed {} to regions {}", topicInfo.topic, isTargetedRegionPush ? targetedRegions : completedDatacenters);
```

Reply:
I thought about ternary operator, but it wont be show if its `targeted` or not. we would need another ternary for it.

Reply:
We don't really need `targeted` IMO.  If we need to tell if that's a targeted push, we can have a complete log ahead of this line, e.g. `LOGGER.info("Starting to push to targeted region XXX`

Reply:
yeah, but when it finished after say 1hr, there will be tons of other logs and you may not easily find the starting log

Reply:
BTW, I think we don't need to pass in the boolean `isTargetedRegionPush`. Just need to check if the `targetedRegions` is empty or not. That would save your work on modifying the method signature in multiple places.

Reply:
Also, can we just drop `targeted `? I think that's okay w/o it, since we have told what regions are pushed in the same log?

Reply:
Synced offline. Resolve this comment now